### PR TITLE
Add Fisherman.

### PR DESCRIPTION
**Pull request by ghost**
Add [Fisherman](http://fisherman.sh).

I think I got it this time. Let me know if this works.

### CODE REVIEW

File: _data/projects/fisherman.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Fisherman
+desc: A blazing fast, modern plugin manager for fish
+site: http://fisherman.sh
+tags:
+- fish-shell
+- shell
```
Comment (shiftkey) [inline _data/projects/fisherman.yml:6]: From the build script:

> /home/travis/build/up-for-grabs/up-for-grabs.net/_data/projects/fisherman.yml - Duplicate tags found: shell, manager, package

Comment (ghost) [inline _data/projects/fisherman.yml:6]: Fixed.

**Comment by shiftkey** [comment]
@bucaran :thumbsup: thanks!

**Comment by ghost** [comment]
Thank you :+1:

**Pull request merged**